libs/components/src/wizard/vl-wizard.component.ts

```diff
@@ -32,13 +32,15 @@ export class VlWizard extends BaseLitElement {
                 attribute: 'data-vl-active-step',
                 reflect: true,
             },
+            showSteps: { type: Boolean, attribute: 'data-vl-show-steps', reflect: true },
```

Comment:
Misschien muggenziften en het zit eigenlijk al fout in de progress bar IMHO. Maar is het eerder "show steps **labels**" oid. De stappen zelf zijn eigenlijk altijd zichtbaar.

Reply:
Ik ben het ermee eens, maar misschien kan deze attr naamswijziging in een ander ticket worden afgehandeld?

Zoals dit PR  gaat over wat de labels tonen in vl-wizard, en een andere PR kan op een ander ticket de naamswijziging van `vl-progress-bar` && `vl-wizard` behandelen, evenals de update van de documentatie?